src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -734,9 +750,12 @@ def _revYangConvert(val):
 
         # if it is a leaf-list do it for each element
         if leafDict[key]['__isleafList']:
-            vValue = list()
-            for v in value:
-                vValue.append(_revYangConvert(v))
+            if tuple(self.elementPath) in self.leaf_list_with_string_value_set:
```

Comment:
if table1 and entry 1 is in set. And table2 and entry2 is in set too. Then (table1, entry2) may create false true, right ? i.e. same leaf name in 2 tables.

Reply:
No. An element in self.leaf_list_with_string_value_set would be like ['PORT', 'Ethernet0', 'speed'], another table with the same leaf name could be ['INTERFACE', 'loopback0', 'speed'], it will not generate false true.